src/functions-and-arrays.js

```diff
@@ -47,6 +138,14 @@ const wordsCount = [
   'matter'
 ];
 
+function howManyTimes(wordsCount, value){
+  var n = 0;
+  for (i = 0; i < wordsCount.length; i++){
+    if(wordsCount[i] == value) {n++}
```

Comment:
Please use strong ("===") comparison. A semicolon behind n++ would not hurt either.